assignments/pig-latin/code.js

```diff
@@ -31,3 +31,15 @@
 //
 // Feel free to play around in the REPL to make sure you understand how this
 // operator works.
+
+const pigLatin = (word) => word.substring(word.search(/[aeiou]/)) + word.substring(0, word.search(/[aeiou]/)) + 'ay'
+
+
+const advancedPigLatin = (word) => {
+  if (word.search(/[aeiou]/) == 0) {
+   return word + 'way'
+  } else {
+    return word.substring(word.search(/[aeiou]/)) + word.substring(0, word.search(/[aeiou]/)) + 'ay'
```

Comment:
This is fine though I'll point out that since this line does the same thing as the basic `pigLatin` function, you could (and arguably) should just call it here:

```suggestion
    return pigLatin(word);
```

This both saves typing and also makes it clear this function either does a special thing if the word starts with a vowel otherwise it does exactly the same thing as the other function.